### PR TITLE
Auto-Compressor Buffs and Manual CPR Tweak

### PR DESCRIPTION
**Pull request by VisVirific**
First, the CPR tweak and the tanks.dm tweak.
* Separates CPR into: Full CPR (Compressions + Mouth-to-Mouth), Compressions only, and Mouth-to-Mouth only
  * Buffs Mouth-to-Mouth in general so it works if you've Lung Rescued someone with a collapsed lung
* Put in a small helper in the tanks.dm so I don't have to keep copy and pasting the same update_gauge() proc

Now, for the Auto-Compressor heckery
* Reworked the Auto-Compressor into a Stabilizer Harness since it doesn't ONLY do automated compressions anymore, now fitted with an Emergency Positive Pressure system like how the IVs have
* To make sure this device remains balanced, it'll now use power from a standard 1k cell, which can be swapped out for better (or worse) ones
* Made the Auto-CPR mode and the EPP system toggeable
* The Auto-CPR mode now has a chance to restart the heart of patients in Asystole/Flat Lining, but it also now has a chance to crack their ribs ONLY IF they're Asystole in the first place

**Tweaked the Auto-Compressor sprites**

**New in-hands and worn icons WITH visible indicators**

**It also has a visible effect for when the Emergency Positive Pressure system is active!**

**Changing the various items with tools**

### CODE REVIEW

**Comment by VisVirific** [comment]
!review

**Comment by NonQueueingMatt** [comment]
The autocompressor already allows people to go into a zombie mode where their BA doesn't fall as is. I'm not sure about adding the ability to restart hearts too. This seems like it'd make them a tad too strong.

File: code/game/objects/items/devices/auto_cpr.dm [hunk 1]
```diff
@@ -51,11 +274,217 @@
...
 		var/obj/item/organ/internal/heart/heart = H.internal_organs_by_name[BP_HEART]
 		if(heart)
-			heart.external_pump = list(world.time, 0.6)
\ No newline at end of file
+			heart.external_pump = list(world.time, 0.6)
+		battery.use(charge_cost)
+
+	if(H.is_asystole())
+		if(H.stat != DEAD && prob(25))
+			H.resuscitate()
```
Comment (NonQueueingMatt) [inline code/game/objects/items/devices/auto_cpr.dm:311]: Lower this probability.

File: code/game/objects/items/devices/auto_cpr.dm [hunk 2]
```diff
@@ -51,11 +274,217 @@
...
+		if(prob(5 * rand(1, 1.5)))
+			var/obj/item/organ/external/chest = H.get_organ(BP_CHEST)
+			if(chest)
+				chest.fracture()
```
Comment (NonQueueingMatt) [inline code/game/objects/items/devices/auto_cpr.dm:315]: Make this probability higher.

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]
These two changes should be all. I'm still unsure about it, but we'll see how it goes. I think the O2 thing should probably be offloaded to a rescue bag, personally, but that's for another day.

**Review by NonQueueingMatt — APPROVED** [review]

**Review by JohnWildkins — APPROVED** [review]

**Pull request merged**